allure-generator/src/main/javascript/plugins/widget-global-variables/GlobalVariablesWidget.hbs

```diff
@@ -0,0 +1,27 @@
+<h2 class="widget__title">
+    {{t 'widget.globalVariables.name'}}
+</h2>
+{{#if items}}
+    <div class="table  table_hover widget__table">
+    {{#each items}}
+        <div class="{{b "table" "row"}}" disabled>
+            <div class="{{b "table" "col"}} long-line">
+                {{name}}
+            </div>
+            <div class="{{b "table" "col"}} long-line">
+                {{#each values}}
+                    {{linky this}}{{#unless @last}},<br>{{/unless}}
+                {{/each}}
+            </div>
+        </div>
+    {{/each}}
+    {{#if overLimit}}
+        <a class="{{b "table" "row"}} clickable globalVariables-widget__expand">
```

Comment:
Class name shouldn't contain upper case letters. Rename to global-variables-widget__expand

Reply:
@skyoker, updated